src/hooks.js

```diff
@@ -59,3 +59,55 @@ exports.useFilter = function (hook, next) {
 
   next();
 };
+
+// Requiring query params
+exports.checkQueryParam = function checkQueryParm(field) {
+  return function(hook, next) {
+    if(hook.params.query[field]) {
+      next();
+    } else {
+      return next(new Error(`Must query with field ${field}`));
+    }
+  }
+}
+
+// Requiring certain fields to exist
+exports.checkRequiredFields = function checkRequiredFields(fields) {
+  return function(hook, next) {
+    let missingFields= [];
```

Comment:
is prettier in this project? 
```suggestion
    let missingFields = [];
```

Reply:
Doesn't look like the project has prettier